autoTickFormat.test.js

```diff
@@ -0,0 +1,43 @@
+import test from 'ava';
+import { autoTickFormatDate, autoTickFormatNumber } from './autoTickFormat';
```

Comment:
Do these functions need to take care of "bad" data like `NaN` or `null`?

Reply:
no they only return a format string to be used by dayjs and numeral. and those are the ones to take care of invalid values. the only data values that go into `autoTickFormat` is the domain, which is expected to be valid numbers or dates.